geopy/distance.py

```diff
@@ -120,6 +119,39 @@
     'GRS-67':        (6378.1600, 6356.774719, 1 / 298.25)
 }
 
+
+def lonlat(x, y, z=0):
+    """
+    Helper function that can be used for more explicit and obvious calling of the distance methods
+    >>> from geopy.distance import lonlat, distance
+    >>> newport_ri = (-71.312796, 41.49008)
+    >>> cleveland_oh = (-81.695391, 41.499498)
+    >>> print(distance(lonlat(*newport_ri), lonlat(*cleveland_oh)).miles)
+    538.3904453677203
+    :param x: longitude 
+    :param y: latitude
+    :param z: altitude
+    :return: Point(latitude, longitude, altitude)
+    """
+    return Point(y, x, z)
+
+
+def latlon(y, x, z=0):
+    """
+    Helper function that can be used for more explicit and obvious calling of the distance methods
+    >>> from geopy.distance import latlon, distance
```

Comment:
I believe that code examples must be indented to be rendered correctly, though I'm not sure. Anyway, all the examples in this file *are* indented, so could you please do that there too?